pandas/core/tools/datetimes.py

```diff
@@ -2,8 +2,14 @@
 import numpy as np
 from collections import MutableMapping
 
-from pandas._libs import lib, tslib
+from pandas._libs import tslib
 from pandas._libs.tslibs.timezones import get_timezone
+from pandas._libs.tslibs import parsing
+from pandas._libs.tslibs.parsing import (  # noqa
+    parse_time_string,
+    _format_is_iso,
+    _DATEUTIL_LEXER_SPLIT,
```

Comment:
I think _DATEUTIL_LEXER_SPLIT Is now private to tstlib.parsing.

Reply:
Just pushed a commit that removes this import.